scripts/neutron-band-structure.py

```diff
@@ -0,0 +1,209 @@
+#! /usr/bin/env python3
```

Comment:
This script also needs to be tested

Reply:
Easier to do if it's in the euphonic namespace, maybe until that is resolved?

Reply:
As long as we don't forget! Lots of PRs/issues on the go currently...